Canadian-Geospatial-Platform/geoview
# Bug fix crash on tab change

# Description

The app would crash when a user would change tab while a data table entry was loading. This is now fixed.

Fixes # (issue)

## Type of change

- [x ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

1. go to http://localhost:8080/outlier-ESRI-maxRecordCount.html
2. go to data table tab
3. click on `advance polling district ...`
4. switch tab immediatly
5. confirm that it no longer crashes

[__Add the URL for your deploy!__](https://pvautour.github.io/geoview/outlier-ESRI-maxRecordCount.html)

# Checklist:

-  ~~I have made corresponding changes to the documentation~~
-  ~~I have added tests that prove my fix is effective or that my feature works~~
-  ~~New and existing unit tests pass locally with my changes~~


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/Canadian-Geospatial-Platform/geoview/2548)
